evap/evaluation/templates/bootstrap_form.html

```diff
@@ -1,4 +1,13 @@
 {% include 'bootstrap_form_errors.html' with errors=form.non_field_errors %}
-{% for field in form %}
-    {% include 'bootstrap_form_field.html' with field=field %}
-{% endfor %}
+{% if email_form and form.html_content in form.visible_fields %}
```

Comment:
would it be possible (without a lot of additional work) to generalize this a bit more? in case we want to use such pill group forms in other places in the future it would be nice if this part isn't specialized on the `email_form`.

Reply:
Is b0ddb08 a fitting solution for that?

Reply:
it's beautiful! 👍